microservices/nris_api/backend/app/commands.py

```diff
@@ -1,15 +1,29 @@
 import click
 
-from app.extensions import db, oracle_db
+from app.extensions import db, sched
+from app.nris.models.nris_raw_data import NRISRawData
+from app.nris.etl.nris_etl import import_nris_xml, etl_nris_data, clean_nris_data, clean_nris_xml_import
+from app.nris.scheduled_jobs import nris_jobs
 
 
 def register_commands(app):
     @app.cli.command()
-    def _test_oracle_db():
-        cursor = oracle_db.cursor()
+    def _get_nris_raw_data():
+        import_nris_xml()
 
-        cursor.execute('select * from CORS.CORS_CV_ASSESSMENTS')
-        col = cursor.fetchone()
-        cursor.close()
-        app.logger.info(col)
-        print(col)
\ No newline at end of file
+    @app.cli.command()
+    def _clean_nris_data():
+        clean_nris_data()
+
+    @app.cli.command()
+    def _clean_nris_raw_data():
+        clean_nris_xml_import()
+
+    @app.cli.command()
+    def _run_nris_etl():
+        etl_nris_data()
+
+    @sched.app.cli.command()
+    def run_nightly_nris_etl():
```

Comment:
Is `run_nightly_nris_etl` identical to `_run_nris_etl ` and exists just to test scheduler?

Reply:
One exists for the scheduler, one exists for manual execution or testing.

Reply:
The above first clears the data and re pulls it, if that succeeds then it truncates the old data and runs the ETL. The _run_nris_etl just runs the ETL it doesn't truncate the tables.

Reply:
```run_nightly_nris_etl``` isn't just for testing, I would say. It lets us use openshift cron to spin up an ETL container with a custom command, rather than having to have our service running all the time.